.github/workflows/ci.yml

```diff
@@ -39,11 +39,12 @@ jobs:
       - name: Install deps
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
+          pip install poetry
```

Comment:
@nllong just fyi if you pip install with poetry: https://python-poetry.org/docs/#alternative-installation-methods-not-recommended. for gh actions, you can also do: https://github.com/NREL/tasty/blob/develop/.github/workflows/ci.yml

Reply:
oh nice. they have a poetry formula! I'll update to that.

Reply:
i've updated this.